content/applications/inventory_and_mrp/inventory/barcode/operations/adjustments.rst

```diff
@@ -1,48 +1,141 @@
+   not need to be scanned. Instead, simply scan the product barcode to start the inventory
+   adjustment.
+
+Alternatively, the quantity can be changed by clicking the :guilabel:`pencil` icon on the far right
+of the product line, and editing the number in the :guilabel:`Quantity` line on the following
+screen. Additionally, the :guilabel:`+1` and :guilabel:`-1` buttons can be clicked to add or
+subtract quantity of the product, and the :guilabel:`number keys` can be used to add quantity, as
+well.
+
+.. example::
+   In the below Inventory Adjustment, the source location `WH/Stock/Shelf/2` was scanned. Then, the
+   barcode for the product `[FURN_7888] Desk Stand with Screen` was scanned 3 times. Additional
+   products can be added to this adjustment by scanning the barcodes for those specific products.
+
+   .. image:: adjustments/adjustments-barcode-inventory-client-action.png
+      :align: center
+      :alt: Barcode Inventory Client Action page with inventory adjustment.
+
+To apply the inventory adjustment, click the green :guilabel:`Apply` button with the check mark at
+the bottom of the page. Once applied, Odoo navigates back to the :guilabel:`Barcode Scanning`
+screen. A small green banner appears in the top right of the page reading: **The inventory
+adjustment has been validated.**
+
+.. admonition:: Did you know?
+
+   Odoo's *Barcode* application provides demo data with barcodes to explore the features of the app.
+   These can be used for testing purposes, and can be printed from the home screen of the app. To
+   access this demo data, navigate to the :menuselection:`Barcode app` and click :guilabel:`stock
+   barcodes sheet` and :guilabel:`commands for Inventory` (bolded and highlighted in blue) in the
+   information pop-up above the scanner.
+
+   .. image:: adjustments/adjustments-barcode-stock-sheets.png
+      :align: center
+      :alt: Demo data prompt pop-up on Barcode app main screen.
+
+Manually add products to inventory adjustment
+=============================================
+
+When creating and applying inventory adjustments in a warehouse, Odoo *Barcode* can also be used to
+manually add products to the adjustment if location and product barcodes are not on hand. To do
+this, first navigate to the :menuselection:`Barcode app`.
+
+Once inside the :guilabel:`Barcode app`, a :guilabel:`Barcode Scanning` screen displaying different
+options is presented. Create a new inventory adjustment by clicking on the :guilabel:`Inventory
+Adjustments` button at the bottom of the screen. This navigates to a new, blank Barcode Inventory
+Client Action page with :guilabel:`Inventory Adjustment` at the top of the form.
+
+To manually add products to this adjustment, click the white :guilabel:`Add Product` button at the
+bottom of the screen.
+
+This navigates to a new, blank page where the product, quantity, and source location can be chosen.
```

Comment:
In order to add a product, I believe all of this info listed must be filled? To my knowledge? 
```suggestion
This opens a new window the product, quantity, and source location must be filled.
```